documents/operating.md

```diff
@@ -387,9 +423,9 @@ The GitHub processes for each decision are outlined in CONTRIBUTING.md and those
 
 - **Changing other sections of `operating.md`:** Plotting Session (needs Strong Consensus).
 
-- **Approving expenses (Preliminary):** GitHub (needs Majority). Preliminary approval of expenses above $100 will allow the Treasure Goblin to release preliminarily approved funds at their discretion within the bounds of the approved expense timeline.  Note: the Treasure Goblin cannot approve their own expenses.
+- **Allocating general operating funds (Preliminary):** GitHub (needs Majority). Preliminary approval of expenses above $100 will allow the Treasure Goblin to release preliminarily approved funds at their discretion within the bounds of the approved expense timeline.  Note: the Treasure Goblin cannot approve their own expenses.
```

Comment:
What happens if an expense is preliminarily allocated but never formally? Is there a required formal approval later? What's the recourse if that vote goes against the allocation? What if it's already been spent?

Reply:
GOOD question...